src/blenderbim/blenderbim/bim/module/pset/__init__.py

```diff
@@ -29,12 +29,19 @@
     operator.GuessQuantity,
     operator.RemovePset,
     operator.TogglePsetExpansion,
+    operator.AddPropertyToEdit,
```

Comment:
Maybe we should start using Blender's recommended operator names ? `BIM_OT_snake_case_op_name`

Reply:
Agreed.